Libraries/dotNetRDF/Storage/StardogConnector.cs

```diff
@@ -2021,19 +2021,23 @@ protected virtual string GetReasoningParameter()
         /// <summary>
         /// Start a transaction
         /// </summary>
+        /// <param name="enableReasoning">Opens the transaction with reasoning enabled.</param>
```

Comment:
As this connector supports older versions of Stardog, I think it would be worth adding a note here that reasoning inside a transaction is only supported in Stardog v5.3.0 onwards.